### DISCUSSION TITLE
Need datetime element

### DISCUSSION THREAD

**Discussion by MuuXB** [zauberzeug/nicegui#2347] [Ideas / Feature Requests]
### Question

1. I want to achieve the effect of selecting both date and time in one input field, like the example shown below(element). Currently, I have checked the official components and they don't seem to meet this requirement. Is there any good solution? 
<img width="546" alt="image" src="https://github.com/zauberzeug/nicegui/assets/33896018/f2cae073-91fa-4f75-b9c0-ebaaa7d07185">

2. Can the style of the official time component be modified? I don't like the rotary style.
<img width="311" alt="image" src="https://github.com/zauberzeug/nicegui/assets/33896018/4005967a-4995-48f1-acb7-2def9595f5a3">

**Comment by falkoschindler** [comment]
Hi @MuuXB, check the [Quasar documentation](https://quasar.dev/vue-components/time) for possible styling options for the QTime component.

For a combined date and time element I'll convert this discussion into a feature request. Any contributions or pull requests are very welcome. It might be possible to create something from `ui.input`, `ui.menu`, `ui.date`, and `ui.time`.
